topics/galaxy-interface/tutorials/upload-data-to-ena/tutorial.md

```diff
@@ -76,7 +74,7 @@ Specifically, we will use one ONT sequencing file to demonstrate interactive met
 
 # Adding ENA Webin credentials to your Galaxy user information
 
-In order to submit data to ENA, you need to have a valid Webin account. If you don't have one already you can register for one [here](https://www.ebi.ac.uk/ena/submit/sra/#registration). Webin credentials need to be included in your Galaxy user information before you can use the {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.3.2) %}.
+In order to submit data to ENA, you need to have a valid Webin account. If you don't have one already you can register for one [here](https://www.ebi.ac.uk/ena/submit/sra/#registration). Webin credentials need to be included in your Galaxy user information before you can use the {% tool [ENA Upload tool](toolshed.g2.bx.psu.edu/repos/iuc/ena_upload/ena_upload/0.6.1) %}.
```

Comment:
I think ALL of these uses (other than the one in the hands-on box) don't need to be the fancy tool links, and it will save you time for future updates. It could just be "you can use the ENA Upload Tool." without any formatting.